clients/fides-js/src/components/Overlay.tsx

```diff
@@ -76,7 +76,7 @@ const Overlay: FunctionComponent<Props> = ({
   const delayBannerMilliseconds = 100;
   const delayModalLinkMilliseconds = 200;
   const hasMounted = useHasMounted();
-  const [bannerIsOpen, setBannerIsOpen] = useState(options.fidesEmbed);
+  const [bannerIsOpen, setBannerIsOpen] = useState(false);
```

Comment:
The reason I did it was because otherwise, when `fides_disable_banner` is `false`, it would flash the modal and then re-render immediately and show the banner. My change didn't take into account `fides_disable_banner` so it should have had the opposite problem when it was set to true. Seems like it should be `options.fidesEmbed && !options.fidesDisableBanner`. Not quite sure why it would break the tests though so I'll do some investigation.

Reply:
So yeah, this is the issue with just setting to to `false`:

![content-flash](https://github.com/ethyca/fides/assets/679632/0d1028e2-8f5d-4e87-a17b-5fce954ef5c3)

Reply:
Okay, I think I understand it now. There's this bit of code in `Overlay.tsx`:

https://github.com/ethyca/fides/blob/df46bf6b55544814c341353b2869d3ff9ff270b5/clients/fides-js/src/components/Overlay.tsx#L144-L150

This is needed to first render `ConsentBanner` in the DOM with `bannerIsOpen=false` and then flip it to `true` after a short delay which will result in the removal of the `fides-banner-hidden` class and a nice entry animation.

In the embed case, we are not animating anything so we can jump to the final state, but it has to be the correct one, otherwise there will be a blip of the other component OR nothing being rendered. Still not sure how this worked before, but I'm glad the e2e test caught it.

Reply:
I think it worked before because we were not rendering the banner at all with the embed mode. But this is a good catch, I've not used the performance tab in this way before, so I learned something new!

Reply:
Ah, gotcha. Alright, I think overall this still feels like a fragile bit of state to manage (ie if we were being extra careful, that "banner is open" state feels like it's the job of the ConsentBanner component itself to handle it's own animation on mount) but this looks good now.